doc/taradino.6.in

```diff
@@ -76,9 +77,11 @@ Maximum time to count down from (next parameter is time in seconds).
 ?
 
 .SH AUTHORS
-ROTT was created and published as shareware by Apogee Software, Ltd. and was published commercially by FormGen, Inc.
+Rise of the Triad was created and published as shareware by Apogee Software, Ltd. and was published commercially by FormGen, Inc.
 .PP
-The primary contributors to the icculus.org port are: Steven Fuller, Ryan C. Gordon, John Hall and Dan Olson.
+The primary contributors to the Taradino port are: Steven Fuller, Ryan C. Gordon, John Hall, Dan Olson, Fabian Greffrath, and erysdren.
```

Comment:
The primary contributors to the original icculus.org port are: ...
The authors of the Taradino port are: ...

Something like this. :wink: